util/create_certs.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+# cp /etc/ssl/openssl.cnf . # copied from OpenSSL 1.1.1b source
```

Comment:
Should we vendor this file (or whatever part is needed for generating our certs), so that the certificate generation works by default?

Reply:
In my opinion, it is not necessary because this script won't be used in near future.

The old cert files were created in 2012.  Because SHA1 is weak nowadays, we need to regenerate the files.  So I guess that the next time to regenerate will be mid or late-2020s.  When we do regenerate, we should fetch the latest version of openssl.cnf at that time.  Moreover, we may need to modify the script itself.  This script is just a memorandum for the work.

Reply:
I see.

I wouldn't ship the script in that case and copy it in the commit message instead since it's about tracking work and not about providing a script that developers are meant to use.

In any case, no big deal, thanks so much for fixing this! :smiley: